pkg/sfs/utils.go

```diff
@@ -103,29 +26,17 @@ func validateCreateVolumeRequest(req *csi.CreateVolumeRequest) error {
 	}
 
 	/*
-	for _, cap := range reqCaps {
-		if cap.GetBlock() != nil {
-			return errors.New("block access type not allowed")
+		for _, cap := range reqCaps {
+			if cap.GetBlock() != nil {
+				return errors.New("block access type not allowed")
+			}
 		}
-	}
 	*/
 
 	return nil
 }
 
-// RoundUpSize calculates how many allocation units are needed to accommodate
-// a volume of given size. E.g. when user wants 1500MiB volume, while AWS EBS
-// allocates volumes in gibibyte-sized chunks,
-// RoundUpSize(1500 * 1024*1024, 1024*1024*1024) returns '2'
-// (2 GiB is the smallest allocatable volume that can hold 1500MiB)
-func RoundUpSize(volumeSizeBytes int64, allocationUnitBytes int64) int64 {
-	roundedUp := volumeSizeBytes / allocationUnitBytes
-	if volumeSizeBytes%allocationUnitBytes > 0 {
-		roundedUp++
-	}
-	return roundedUp
-}
-
+// TODO after feat-sfs-idempotent merged. Delete this method
```

Comment:
do you mean remove `bytesToGiB`?  i see it is used in controller

Reply:
done